docs/user/software/development/java.md

```diff
@@ -1,34 +1,40 @@
 ---
 title: Java
-summary: A quick guide to Java development on Solus
+summary: Learn how to install, run, and manage Java on Solus.
 ---
 
 # Java
 
-This article covers obtaining open source Java (otherwise known as OpenJDK). In this article, two abbreviations will be used to refer to components of the Java software suite:
+Solus includes multiple LTS versions of Java in the repositories. You can install more than one version in your system at once.
 
-- JRE (Java Runtime Environment)
-- JDK (Java Development Kit)
+:::tip
 
-## Installing OpenJDK
+If you need to install different versions or Java, use alternative installation methods such as _SDKMAN!_.
 
-Solus currently offers two Java versions in the repository, and these versions can be installed alongside one another. For ease of maintenance, Solus only includes LTS versions of OpenJDK. Those who require the absolute newest version of the JDK, or other non-LTS versions, should look to other methods of installation (such as [sdkman](https://sdkman.io/)).
+:::
 
-### OpenJDK 11
+All Java packages in the Solus repositories include the _Java Runtime Environment_ (JRE) and the _Java Development Kit_ (JDK).
```

Comment:
Is there a reason for emphasizing these names?

Reply:
Style guides such as the [Google Style Guide](https://developers.google.com/style/key-terms) recommend using emphasis tags when introducing new terms or concepts. 

In this case, since I'm defining the meaning of both JRE and SDK, I opted to emphasize the first occurrence of the terms.